spec/views/records/_form.html.erb_spec.rb

```diff
@@ -6,8 +6,19 @@
   end
 
   let(:resource) do
-    ValidatedAssetRepository.new(Image).new
+    Reviewable.new(ValidatedAssetRepository.new(Image).new)
   end
+
+  let(:saved_resource) do
+    resource.tap do |r|
+      r.save
+    end
+  end
+
+  let(:saved_resource_form) do
+    ImageForm.new(saved_resource)
```

Comment:
I can do that. I did that in my other PR for some other method, but didn't know if that was acceptable or not.